test/plots/rectanglePlotTests.ts

```diff
@@ -2,511 +2,486 @@
+        plot.attr("fill", (d) => d.magnitude, colorScale);
+        plot.x((d: any) => d.x, xScale);
+        plot.y((d: any) => d.y, yScale);
+        plot.renderTo(svg);
+        let data = [
+          {x: "A", y: "W", magnitude: 0},
+          {x: "B", y: "X", magnitude: 8},
+          {x: "C", y: "Y", magnitude: 16},
+          {x: "D", y: "Z", magnitude: 24}
+        ];
+        dataset.data(data);
+        let cells = plot.content().selectAll("rect")[0];
+        assert.strictEqual(cells.length, data.length);
+        for (let i = 0; i < cells.length; i++) {
+          let cell = d3.select(cells[i]);
+          assert.strictEqual(cell.attr("x"), String(i * CELL_WIDTH), "Cell x coord is correct");
+          assert.strictEqual(cell.attr("y"), String(i * CELL_HEIGHT), "Cell y coord is correct");
+          assert.strictEqual(cell.attr("width"), String(CELL_WIDTH), "Cell width is correct");
+          assert.strictEqual(cell.attr("height"), String(CELL_HEIGHT), "Cell height is correct");
+        }
+        svg.remove();
+      });
 
-    it("renders correctly when data is set after construction", () => {
-      let xScale = new Plottable.Scales.Category();
-      let yScale = new Plottable.Scales.Category();
-      let colorScale = new Plottable.Scales.InterpolatedColor();
-      colorScale.range(["black", "white"]);
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let dataset = new Plottable.Dataset();
-      let gridPlot = new Plottable.Plots.Rectangle();
-      gridPlot.addDataset(dataset)
-              .attr("fill", (d) => d.magnitude, colorScale);
-      gridPlot.x((d: any) => d.x, xScale)
-              .y((d: any) => d.y, yScale)
-              .renderTo(svg);
-      dataset.data(DATA);
-      VERIFY_CELLS((<any> gridPlot)._renderArea.selectAll("rect")[0]);
-      svg.remove();
+      it("can invert y axis correctly", () => {
+        let plot = new Plottable.Plots.Rectangle();
+        plot.addDataset(new Plottable.Dataset(data));
+        plot.attr("fill", (d) => d.magnitude, colorScale);
+        plot.x((d: any) => d.x, xScale);
+        plot.y((d: any) => d.y, yScale);
+        plot.renderTo(svg);
+
+        yScale.domain(["U", "V"]);
+
+        let cells = plot.content().selectAll("rect")[0];
+        let cellAU = d3.select(cells[0]);
+        let cellAV = d3.select(cells[2]);
+        cellAU.attr("fill", "#000000");
+        cellAU.attr("x", "0");
+        cellAU.attr("y", "100");
+
+        cellAV.attr("fill", "#ffffff");
+        cellAV.attr("x", "0");
+        cellAV.attr("y", "0");
+
+        yScale.domain(["V", "U"]);
+        cells = plot.content().selectAll("rect")[0];
+        cellAU = d3.select(cells[0]);
+        cellAV = d3.select(cells[2]);
+        cellAU.attr("fill", "#000000");
+        cellAU.attr("x", "0");
+        cellAU.attr("y", "0");
+
+        cellAV.attr("fill", "#ffffff");
+        cellAV.attr("x", "0");
+        cellAV.attr("y", "100");
+        svg.remove();
+      });
```

Comment:
There are a ton of attribute setters here with 0 assertions being made.  What is this test doing here?